packages/foreman/foreman/foreman.spec

```diff
@@ -758,6 +759,7 @@ install -Dp -m0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/cron.d/%{name}
 install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
+install -Dp -m0755 extras/%{name}-sidekiq %{buildroot}%{_datadir}/%{name}/bin/%{name}-sidekiq
```

Comment:
On line 765 we copy the entire bin directory. This makes the line redundant if you keep it in `bin/` in foreman.git.

Reply:
Oh good catch. I made sure attributes are correct.